### PR TITLE
Update containerd to version 1.6.10

### PR DESCRIPTION
**Pull request by dimitar-dimitrow**
[#88] Update containerd to version 1.6.10

- bump containerd version
- update NOTICE file

Signed-off-by: Dimitar Dimitrov <dimitar.dimitrov3@bosch.io>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -72,25 +69,26 @@ require (
...
-	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae // indirect
+	github.com/vishvananda/netns v0.0.0-20210104183010-2eb08e3e575f // indirect
 	github.com/yusufpapurcu/wmi v1.2.2 // indirect
 	go.mozilla.org/pkcs7 v0.0.0-20200128120323-432b2356ecb1 // indirect
-	go.opencensus.io v0.22.3 // indirect
+	go.opencensus.io v0.23.0 // indirect
 	golang.org/x/mod v0.4.2 // indirect
-	golang.org/x/net v0.0.0-20211216030914-fe4d6282115f // indirect
-	golang.org/x/term v0.0.0-20210615171337-6886f2dfbf5b // indirect
+	golang.org/x/net v0.0.0-20220722155237-a158d28d115b // indirect
```
Comment (e-grigorov) [inline go.mod:85]: I cannot find this dependency in the notice file. Could you please double check whether it's a runtime dependency or not?

File: go.mod [hunk 2]
```diff
@@ -72,25 +69,26 @@ require (
...
+	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211 // indirect
```
Comment (e-grigorov) [inline go.mod:86]: I cannot find this dependency in the notice file. Could you please double check whether it's a runtime dependency or not?

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (dimitar-dimitrow) [reply go.mod:85]: It is there under golang/net. See https://github.com/eclipse-kanto/container-management/blob/1c125110107caa9975af51c2d878b7e73f5b52c1/NOTICE.md?plain=1#L405

File: go.mod (hunk 2 above)
Comment (dimitar-dimitrow) [reply go.mod:86]: It is there under golang/term. See https://github.com/eclipse-kanto/container-management/blob/1c125110107caa9975af51c2d878b7e73f5b52c1/NOTICE.md?plain=1#L423

**Review by e-grigorov — APPROVED** [review]

**Review by konstantina-gramatova — APPROVED** [review]

**Pull request merged**